hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -112,3 +112,26 @@ db_username: null
 db_password: null
 db_name: null
 
+# Asset Restriction
+asset_restriction_enabled: false
+asset_restriction:
+- - RLC
```

Comment:
Do these values apply as defaults when the bot first loads this template?
We shouldn't have any default values for this parameters, I think.

Reply:
These values are just a template to populate the list; I reset my client and the values are set to the appropriate default:
![Screen Shot 2020-06-11 at 1 40 46 AM](https://user-images.githubusercontent.com/20906874/84349094-97db8400-ab84-11ea-9b0b-0253600bd0c4.png)